Jenkinsfile

```diff
@@ -16,6 +16,7 @@ node ('!(ci-agent-4)') {
     stage('Installing Packages for python 3 release') {
       sh("rm -rf ./venv")
       sh("python3.6 -m venv ./venv")
+      sh("bash -c 'venv/bin/python -m pip install --upgrade 'pip==20.3.4''")
```

Comment:
And here?

Reply:
Added a commit message to explain the need for an upgrade